libs/SmartSync/SmartSync/Classes/Util/SFSyncState.h

```diff
@@ -138,6 +142,11 @@ extern NSString * const kSFSyncStateMergeModeLeaveIfChanged;
 + (SFSyncStateMergeMode) mergeModeFromString:(NSString*)mergeMode;
 + (NSString*) mergeModeToString:(SFSyncStateMergeMode)mergeMode;
 
+/** Setter and getter of property errorJSON
+ */
+- (void) setErrorJSON:(NSString * _Nonnull)errorJSON;
```

Comment:
Do we need a getter/setter?

Reply:
Resolved in commit: 3ebdacfc3fd423b87501bc0da3b856778c0b173c
But the setter is still needed for setting error into the sync object and the setter interface is still required by compiler.